src/metabase/automagic_dashboards/dashboard_templates.clj

```diff
@@ -286,6 +287,20 @@
   [dashboard-template]
   (transduce (map (comp count ancestors)) + (:applies_to dashboard-template)))
 
+(defn- set-default-card-dimensions
+  "Update the card template dimensions to align with the default FE dimensions."
+  [dashboard-template]
+  (update dashboard-template :cards
+          (fn [cards]
+            (mapv
+              (fn [card-spec]
+                (update-vals
+                  card-spec
+                  (fn [{:keys [visualization] :as card-spec}]
+                    (let [defaults (get-in magic.constants/card-size-defaults [(keyword visualization) :default])]
+                      (into defaults card-spec)))))
```

Comment:
This function would be a lot easier to read if this were a named helper. Then the outer function becomes `(update dashboard-template :cards #(mapv new-helper %))`.

Reply:
Done